.eslintrc.js

```diff
@@ -106,6 +106,13 @@ module.exports = {
     //
 
     curly: ['error', 'all'],
+    eqeqeq: [
+      'error',
+      'always',
+      {
+        null: 'ignore',
```

Comment:
Instead of ignore should we should enforce one way or the other.

I've mentioned that I personally think that using non-strict equality is best and never ever write the undefined variable. I don't think there are any cases in our codebase where we would need or want to treat undefined and null separately.

Reply:
Personally I find it much more easy to have never "==" in my code base. Nothing to think about. 
But its your code base :-)